audit.json

```diff
@@ -28,5 +28,15 @@
   "10010_Cookie No HttpOnly Flag_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?epimms_id=epimms_id_GET" : "ignore",
   "10054_Cookie without SameSite Attribute_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?epimms_id=epimms_id_GET" : "ignore",
   "10096_Timestamp Disclosure - Unix_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?epimms_id=epimms_id_GET" : "ignore",
-  "100000_A Client Error response code was returned by the server_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?epimms_id=epimms_id_GET" : "ignore"
+  "100000_A Client Error response code was returned by the server_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?epimms_id=epimms_id_GET" : "ignore",
+  "10010_Cookie No HttpOnly Flag_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?building_location_name=building_location_name&epimms_id=epimms_id_GET" : "ignore",
+  "10010_Cookie No HttpOnly Flag_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/region?region=region_GET" : "ignore",
+  "10054_Cookie without SameSite Attribute_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?building_location_name=building_location_name&epimms_id=epimms_id_GET" : "ignore",
+  "10054_Cookie without SameSite Attribute_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/region?region=region_GET" : "ignore",
+  "100000_A Client Error response code was returned by the server_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?building_location_name=building_location_name&epimms_id=epimms_id_GET" : "ignore",
+  "100000_A Client Error response code was returned by the server_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/region?region=region_GET" : "ignore",
+  "90033_Loosely Scoped Cookie_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?building_location_name=building_location_name&epimms_id=epimms_id_GET" : "ignore",
+  "90033_Loosely Scoped Cookie_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/region?region=region_GET" : "ignore",
+  "10096_Timestamp Disclosure - Unix_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/building-locations?building_location_name=building_location_name&epimms_id=epimms_id_GET" : "ignore",
+  "10096_Timestamp Disclosure - Unix_http://rd-location-ref-api-aat.service.core-compute-aat.internal/refdata/location/region?region=region_GET" : "ignore"
```

Comment:
Are these exclusions required?

Reply:
Yes Vijay, it is needed, otherwise build won't be passed.